design/block_excange_format.md

```diff
@@ -6,11 +6,53 @@ serialize, deserialize, import, and export SeedBlock programs.
 A JSON object with the block exchange format represents a single SeedBlock code
```

Comment:
Typo in the file name "block_excange_format.md".

Reply:
Done.